source/common/common/assert.h

```diff
@@ -16,6 +16,21 @@ namespace Envoy {
     }                                                                                              \
   } while (false)
 
+/**
+ * A version of RELEASE_ASSERT that allows for ostream style extra data.
+ */
+#define VERBOSE_RELEASE_ASSERT(X, Y)                                                               \
```

Comment:
I think we want something closer to what happens in `gtests`, e.g. `ASSERT_EQ(foo, bar) << baz;`